pom.xml

```diff
@@ -77,21 +78,6 @@
       <artifactId>guava</artifactId>
       <version>29.0-jre</version>
     </dependency>
-    <dependency>
-      <groupId>com.github.cbioportal.cbioportal</groupId>
-      <artifactId>maf</artifactId>
-      <version>ee5802d836c05ed846d7d1ea3f584febdc07ffa8</version>
-      <exclusions>
-        <exclusion>
-          <groupId>ch.qos.logback</groupId>
-          <artifactId>logback-core</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>ch.qos.logback</groupId>
-          <artifactId>logback-classic</artifactId>
-        </exclusion>
-      </exclusions>
-    </dependency>
```

Comment:
@sheridancbio do we still want to keep this?

Reply:
Yes, for now I think we should keep this dependency. @haynescd pointed out that maf is a dependency in cbioportal-core, and since the maf code is used in both this repository and in that one we can keep it in a separate repository I think.

Hopefully we can develop a unified community-wide tool for study import which includes the ability to annotate variants (via genome nexus). That would essentially replace cbioportal-core. At that time we could decide whether the ability to parse and represent variants from a mutation file should be its own module or become embedded in one of these two repositories (most likely this one).

Reply:
Note that we still do have the dependency .. it is simply moved to the annotator module (it is only needed there)